server/routes.go

```diff
@@ -1190,11 +1192,11 @@ func Serve(ln net.Listener) error {
 		srvr.Close()
 		schedDone()
 		sched.unloadAllRunners()
-		gpu.Cleanup()
+		payloads.Cleanup(payloadFS)
```

Comment:
Perhaps for the future: do we want to clean this up always? Vs in the background on first start?

Reply:
The payloads can be large on a linux host, so we should probably not leave cruft lying around when we're not running.